package.json

```diff
@@ -95,10 +95,10 @@
     "stylelint": "^10.0.1",
     "stylelint-config-terra": "^3.0.0",
     "terra-aggregate-translations": "^1.0.0",
-    "terra-dev-site": "^5.0.0",
+    "terra-dev-site": "cerner/terra-dev-site#update-axe-rules",
```

Comment:
What is this dark magic.

Reply:
https://medium.com/@jonchurch/use-github-branch-as-dependency-in-package-json-5eb609c81f1a

Reply:
![giphy](https://user-images.githubusercontent.com/6720991/59528798-d91f5200-8ea4-11e9-8990-5e8b6f8d9043.gif)